src/routes/Profile/BadgeCard/BadgeCard.tsx

```diff
@@ -27,53 +29,70 @@ export type BadgeCardProps = {
   disableClaiming: boolean;
 };
 
-export const BadgeCard: React.FunctionComponent<BadgeCardProps> = ({
-  onClaimButtonClick,
-  claimButtonMode,
-  loading,
-  variant,
-  claimedAt,
-  disableClaiming,
-}) => {
-  return (
-    <Container data-testid="BadgeCard">
-      <BadgePillBox>
-        <BadgePill loading={loading} variant={variant} />
-      </BadgePillBox>
-      <BadgeBox>
-        <Badge loading={loading} variant={variant} />
-      </BadgeBox>
-      <TitleBox>
-        {loading ? (
-          <TitleSkeleton variant="text" />
-        ) : (
-          <TitleTypography variant="body2">
-            {BADGE_VARIANT_TITLE_COPY_MAP[variant].toUpperCase()}
-          </TitleTypography>
-        )}
-      </TitleBox>
-      <DescriptionBox>
-        {loading ? (
-          <DescriptionSkeleton variant="text" />
-        ) : (
-          <DescriptionTypography variant="body2">
-            {BADGE_VARIANT_DESCRIPTION_COPY_MAP[variant]}
-          </DescriptionTypography>
-        )}
-      </DescriptionBox>
-      {disableClaiming ? null : (
-        <ClaimButtonBox>
+export type BadgeCardHandle = {
+  scrollIntoView: () => void;
+};
+
+export const BadgeCard = forwardRef<BadgeCardHandle, BadgeCardProps>(
+  ({ onClaimButtonClick, claimButtonMode, loading, variant, claimedAt, disableClaiming }, ref) => {
+    const containerRef = useRef<HTMLElement>(null);
```

Comment:
can you explain how this reference works and what's it's utility

Reply:
We need a ref to the div element so that you can later on use it like: containerRef.current?.scrollIntoView. Basically when we want the scrollbar to position it self on top of div.

Reply:
got it, thanks